UnityProject/Assets/Scripts/Core/Chat/DiscordWebhookMessage.cs

```diff
@@ -131,15 +131,6 @@ private async void Post(string url, JsonPayloadContent playload)
 			var content = new StringContent(jsonContent, Encoding.UTF8, "application/json");
 
 			HttpResponseMessage response = await SafeHttpRequest.PostAsync(url, content);
-
-			if (response.IsSuccessStatusCode)
-			{
-				Logger.Log("Request was successful.");
-			}
-			else
```

Comment:
we probably want to log the failed request, as an error

Reply:
ok